tests/wallet3_tests/mock_daemon_comms.hpp

```diff
@@ -11,6 +11,8 @@ class MockDaemonComms: public DefaultDaemonComms
 
     MockDaemonComms() : DefaultDaemonComms(get_omq()){};
```

Comment:
this should probably subclass the DaemonComms interface class rather than the concrete implementation.